FJ-about-smoke-spec.js

```diff
@@ -1,19 +1,58 @@
-describe('The new consumerfinance.gov', function() {
+describe( 'Beta Contact Page', function() {
+  var passedURL = browser.params.base_url;
+  var baseURL = passedURL ? passedURL : 'http://beta.consumerfinance.gov/';
+  var contactURL = baseURL + 'contact-us/';
+  var phoneClass = 'list_link__phone';
 
-  beforeEach(function() {
+  beforeEach( function() {
     return browser.ignoreSynchronization = true;
-  });
+  } );
 
-  it('should properly load in a browser', function(){
-    browser.get('http://beta.consumerfinance.gov/'); // This could be set to a staging server or localhost.
-    expect(browser.getTitle()).toBe('Consumer Financial Protection Bureau');
-  });
+  it( 'should properly load in a browser', function(){
+    browser.get( contactURL ); // This could be set to a staging server or localhost.
+    expect( browser.getTitle() ).toBe( 'Contact us' );
+  } );
 
-  it('should show the correct phone number', function(){
-    browser.get('http://beta.consumerfinance.gov/contact-us');
-    var phoneNumber = '(855) 411-CFPB (2372)'
-    var phoneElement = element(by.partialLinkText(phoneNumber));
-    expect(phoneElement.getText()).toBeDefined();
-  });
+  it( 'should include the Submit a Complaint phone numbers', function(){
+    var phone = element.all( by.partialLinkText( '(2372)' ) );
+    var firstPhone = phone.first();
+    var secondPhone = phone.last();
 
-});
+    expect( phone.count() ).toEqual( 2 );
+    expect( phone.getAttribute( 'class' ) ).toMatch( phoneClass );
+    expect( firstPhone.getText() ).toBe( '(855) 411-CFPB (2372)' );
+    expect( firstPhone.getAttribute( 'href' ) ).toBe( 'tel:8554112372' )
+    expect( secondPhone.getText() ).toBe( '(855) 729-CFPB (2372) TTY' );
+    expect( secondPhone.getAttribute( 'href' ) ).toBe( 'tel:8557292372' );
+  } );
+
+/* This hasn't been released to beta yet
```

Comment:
Instead of commenting this out use `xit` on L:30 in place of `it` to set it as a pending test (should the test suite support that syntax).